web/m00n89/html/attributes/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="it" class="translated-ltr">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Discover...</title>
+  </head>
+  <body>
+    <h1 style="text-align:center;">HTML</h1>
+    <h2 style="color: cadetblue;">Riassunto capitolo</h2>
+    <p>All HTML elements can have attributes: 
+      <br>
+      The href attribute of <a> specifies the URL of the page the link goes to <br>
+      The src attribute of <img> specifies the path to the image to be displayed <br>
```

Comment:
I don't see any image.
Try with `<img src="path-to-the-image.jpg" alt="A man standing next to a clothes shop" />`